src/test/java/com/hm/achievement/db/SQLiteDatabaseBasicTest.java

```diff
@@ -0,0 +1,220 @@
+package com.hm.achievement.db;
+
+import com.hm.achievement.AdvancedAchievements;
+import com.hm.achievement.exception.PluginLoadError;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import utilities.MockUtility;
+
+import java.sql.Date;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+/**
+ * Class for testing SQLite Database.
+ *
+ * @author Rsl1122
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class SQLiteDatabaseBasicTest extends SQLiteDatabaseTest {
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        MockUtility mockUtility = MockUtility.setUp()
+                .mockLogger()
+                .mockPluginConfig();
+        AdvancedAchievements pluginMock = mockUtility.getPluginMock();
+
+        db = new SQLiteDatabaseManager(pluginMock);
+        initDB();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        clearDatabase();
+    }
+
+    @AfterClass
+    public static void tearDownClass() {
+        if (db != null) {
+            db.shutdown();
+        }
+    }
+
+    @Test
+    public void testInitialization() throws PluginLoadError {
+        initDB();
+    }
+
+    @Test
+    public void testGetAchievementList() {
+        registerAchievement();
+        sleep100ms();
+
+        List<String> achievements = db.getPlayerAchievementsList(testUUID);
+        assertEquals(3, achievements.size());
+
+        List<String> expected = Arrays.asList(testAchievement, testAchievementMsg, achievements.get(2));
+        assertEquals(expected, achievements);
+    }
+
+    @Test
+    public void testAchievementCount() {
+        registerAchievement();
+        sleep100ms();
+
+        Map<UUID, Integer> expected = Collections.singletonMap(testUUID, 1);
+
+        Map<UUID, Integer> actual = db.getPlayersAchievementsAmount();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void testAchievementDateRegistration() {
+        String date = db.getPlayerAchievementDate(testUUID, testAchievement);
+        assertNull(date);
+
+        registerAchievement();
+        sleep100ms();
+
+        date = db.getPlayerAchievementDate(testUUID, testAchievement);
+        assertNotNull(date);
+    }
+
+    @Test
+    public void testPlayerAchievementAmount() {
+        registerAchievement();
+        sleep100ms();
+
+        assertEquals(1, db.getPlayerAchievementsAmount(testUUID));
+    }
+
+    @Test
+    public void testDeleteAchievement() throws PluginLoadError {
```

Comment:
`throws PluginLoadError` no longer needed here.

Reply:
Seems like IntelliJ code inspection missed that one